scripts/setup.bitcoin-node.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Initialize a flag for the --reset option
+reset_flag=false
+
+# Loop through the arguments
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+  --reset)
```

Comment:
Support for a `--help` flag is always welcome.

Reply:
Good idea, done!